### PR TITLE
[MFTF] Adding AdminFillTaxIdentifierFilterOnTaxRateGridActionGroup

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
AdminFillTaxIdentifierFilterOnTaxRateGridActionGroup has been added
Tests have been updated with AdminFillTaxIdentifierFilterOnTaxRateGridActionGroup

### CODE REVIEW

**Comment by AnnaAPak** [comment]
magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE,Functional Tests EE,Functional Tests B2B

**Comment by eduard13** [comment]
@magento run all tests

File: app/code/Magento/Tax/Test/Mftf/Test/UpdateLargeTaxRateEntityTest.xml [hunk 1]
```diff
@@ -44,8 +46,9 @@
         <!-- Verify we see updated large tax rate(from the above step) on the tax rate grid page -->
         <actionGroup ref="AdminTaxRateGridOpenPageActionGroup" stepKey="goToTaxRateIndex4"/>
         <click selector="{{AdminDataGridHeaderSection.clearFilters}}" stepKey="clickClearFilters2"/>
-        <fillField selector="{{AdminTaxRateGridSection.filterByTaxIdentifier}}" userInput="{{defaultTaxRateWithLargeRate.code}}" stepKey="fillTaxIdentifierField3"/>
-        <click selector="{{AdminTaxRateGridSection.search}}" stepKey="clickSearch2"/>
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/Test/UpdateLargeTaxRateEntityTest.xml:48]: We should preserve the `clickSearch2` reference, this is the reason of the SVC failing.

**Comment by AnnaAPak** [comment]
@magento run all tests

**Review by eduard13 — APPROVED** [review]
The failing tests aren't related to these changes.

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-8594](https://jira.corp.magento.com/browse/ENGCOM-8594) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Review by gabrieldagama — CHANGES_REQUESTED** [review]
Hi @AnnaAPak, can you have a look at the conflicts? Thank you!

**Comment by AnnaAPak** [comment]
@magento run all tests

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by eduard13** [comment]
@magento run Functional Tests B2B , Functional Tests CE, Functional Tests EE

**Review by eduard13 — APPROVED** [review]
The failed tests don't seem to be related to the failed functional tests

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-8594](https://jira.corp.magento.com/browse/ENGCOM-8594) has been created to process this Pull Request

**Pull request merged**